adafruit_platformdetect/board.py

```diff
@@ -202,6 +206,10 @@ def _beaglebone_id(self):
         return None
     # pylint: enable=no-self-use
 
+    def _apq8016_id(self):
```

Comment:
It would be better to create an array of board IDs for 96Boards and just check whether the detected board is of any kind and return its board ID (like how Rpi is doing). This will help in maintaining different 96Boards in future.

Reply:
i dont think we want/need to detect a variety of 96boards for circuitpython, because we have to define each board/chip low-speed-gpio-pinout and even if they're 'pretty much the same' in functionality, each chip is going to have unique peripherals. so a separate `is_96board` property would be best, and it can test the various 96boards that are supported :)

Reply:
@ladyada So I have updated for it to only detect as 96Boards and not as separate boards, also added a couple more boards as an example. Chip detection still works as is.
Our problem with doing the way rpi or bb has been done is that they use SoCs from the same family BCM2XXX or AM33XX.
We have a variety of vendors. Also atm we don't have a common tag/field in the device tree that might tell that a given board is a part of 96Boards or not.
As you can see we can still add Board specific id using `_is_96Boards` if required.